fang/README.md

```diff
@@ -31,32 +31,62 @@ Here are some of the fang's key features:
 
 ```toml
 [dependencies]
-fang = { version = "0.10.4" , features = ["blocking"], default-features = false }
+fang = { version = "0.11.0" , features = ["blocking"], default-features = false }
 ```
 
 #### the Asynk feature
 
+- PostgreSQL as a queue
+
+```toml
+[dependencies]
+fang = { version = "0.11.0" , features = ["asynk-postgres"], default-features = false }
+```
+
+- SQLite as a queue
+
+```toml
+[dependencies]
+fang = { version = "0.11.0" , features = ["asynk-sqlite"], default-features = false }
+```
+
+- MySQL as a queue
+
 ```toml
 [dependencies]
-fang = { version = "0.10.4" , features = ["asynk"], default-features = false }
+fang = { version = "0.11.0" , features = ["asynk-mysql"], default-features = false }
 ```
 
 #### the Asynk feature with derive macro
 
+Substitute `database` with your desired backend.
+
 ```toml
 [dependencies]
-fang = { version = "0.10.4" , features = ["asynk", "derive-error" ], default-features = false }
+fang = { version = "0.11.0" , features = ["asynk-{database}", "derive-error" ], default-features = false }
 ```
 
 #### All features
```

Comment:
would be nice to list all features and describe them

Reply:
it is explained in the lines before

https://github.com/ayrat555/fang/pull/141/files/108d55c7208b2b44d90e12baad1a635a724ff259#diff-48a5c39d84ad1a541e05b07a5ad02ec80ceef510a3c66955aab6a77485e132b3R39-R57